www/modules/migrate/services/google-calendar.client.service.es6

```diff
@@ -1,33 +1,33 @@
 class GoogleCalendarService {
-    /** @ngInject */
-    constructor($http) {
-        Object.assign(this, {
-            $http
-        });
-    }
+  /** @ngInject */
+  constructor($http) {
+    Object.assign(this, {
+      $http
+    });
+  }
 
-    getAuthUrl() {
-        return this.$http.get('migrate/google-calendar/get-auth-url').then(responce => responce.data);
-    }
+  getAuthUrl() {
+    return this.$http.get('migrate/google-calendar/get-auth-url').then(responce => responce.data);
+  }
 
-    getCalendarEvents(params) {
-        return this.$http.get('migrate/google-calendar/get-calendar-events', {params}).then(responce => responce.data);
-    }
+  getCalendarEvents(params) {
+    return this.$http.get('migrate/google-calendar/get-calendar-events', { params }).then(responce => responce.data);
+  }
 
-    convertCalendarEvents(events) {
-        return events.map(event => {
-            return {
-                id: event.id,
-                type: 'event',
-                title: event.summary,
-                notes: event.description,
-                days: {
-                    startTime: event.start.dateTime,
-                    endTime: event.end.dateTime
-                }
-            }
-        });
-    }
+  convertCalendarEvents(events) {
+    return events.map(event => {
+      return {
+        id:    event.id,
+        type:  'event',
+        title: event.summary,
+        notes: event.description,
+        days:  {
+          startTime: event.start.dateTime,
+          endTime:   event.end.dateTime
```

Comment:
Could you please double check whether it's `dateTime` or just `time` comes from google-calendar API